.github/workflows/docker.yml

```
@@ -34,12 +34,12 @@ jobs:
         arch:
           - amd64
           - arm64
-    
+
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version: '1.21'
+          go-version-file: go.mod
```

Perhaps also incldue cache-dependency-path:

```suggestion
          go-version-file: go.mod
          cache-dependency-path: go.sum
```